src/composables/use-filter-sidebar-visibility.js

```diff
@@ -0,0 +1,28 @@
+import { onMounted, readonly, ref } from '@nuxtjs/composition-api'
+import local from '~/utils/local'
+
+const isFilterSidebarVisible = ref(false)
+
+export function useFilterSidebarVisibility({ mediaQuery }) {
+  /**
+   * Open or close the filter sidebar
+   * @param {boolean} val
+   */
+  const setFilterSidebarVisibility = (val) => {
+    isFilterSidebarVisible.value = val
+    local.set(process.env.filterStorageKey, val)
+  }
+
+  onMounted(() => {
+    const localFilterState = () =>
+      local.get(process.env.filterStorageKey)
+        ? local.get(process.env.filterStorageKey) === 'true'
+        : false
```

Comment:
This can be simplified:

```suggestion
      local.get(process.env.filterStorageKey) === 'true'
```